app/controllers/participants/validations_controller.rb

```diff
@@ -52,26 +52,129 @@ def cannot_find_details
 
     def complete
       @school = participant_profile.school
+
+      partial_name =
+        if participant_profile.sit_mentor? || !FeatureFlag.active?(:eligibility_notifications)
+          nil
+        elsif participant_profile.school_cohort.full_induction_programme?
+          if participant_profile.ect?
+            fip_ect_completed_page
+          else
+            fip_mentor_completed_page
+          end
+        elsif participant_profile.school_cohort.core_induction_programme?
+          if participant_profile.ect?
+            cip_ect_completed_page
+          else
+            cip_mentor_completed_page
+          end
+        end
+
+      if partial_name.present?
+        render partial_name
+      else
+        render_default_completed_page
+      end
+    end
+
+  private
+
+    def fip_ect_completed_page
+      @partnership = @school.partnerships.active.find_by(cohort: participant_profile.school_cohort.cohort)
+
+      if participant_profile.ecf_participant_eligibility&.eligible_status?
+        if @partnership.present?
+          "fip_eligible_partnership"
+        else
+          "fip_eligible_no_partnership"
+        end
+      elsif participant_profile.ecf_participant_eligibility.blank? || participant_profile.ecf_participant_eligibility.different_trn_reason?
+        if @partnership.present?
+          "fip_no_trn_match_partnership"
+        else
+          "fip_no_trn_match_no_partnership"
+        end
+      elsif participant_profile.ecf_participant_eligibility.manual_check_status? ||
+          participant_profile.ecf_participant_eligibility.ineligible_status?
+
+        case participant_profile.ecf_participant_eligibility.reason
+        when "previous_induction"
+          "fip_ect_previous_induction"
+        when "active_flags"
+          if participant_profile.ecf_participant_eligibility.manual_check_status?
+            "fip_active_flags_manual_check"
+          else
+            "fip_active_flags_ineligible"
+          end
+        when "no_qts"
+          "fip_ect_no_qts"
+        end
+      end
+    end
+
+    def fip_mentor_completed_page
```

Comment:
To be honest, there were originally more differences than just the previous induction/previous participation in here originally before it changed. I'm still not certain there won't be more content changes coming out of testing. Since this is just repeated twice, I don't think there will be much benefit in DRYing this up, more likely to create a single bigger, more confusing method.